dockerfiles/Dockerfile-coda-daemon-puppeteered

```diff
@@ -23,7 +23,8 @@ while [ -f daemon-active ]; do sleep 1; done'\
 
 RUN chmod +x find_puppeteer.sh start.sh stop.sh
 
-RUN curl -sL "https://raw.githubusercontent.com/MinaProtocol/mina/${CODA_BRANCH}/dockerfiles/coda_daemon_puppeteer.py" \
-  -o /root/coda_daemon_puppeteer.py
+ADD coda_daemon_puppeteer.py /root/coda_daemon_puppeteer.py
```

Comment:
wasn't there a reason we were grabbing the py file from github rather than simply ADD-ing it into the container?  although i suppose if this works, then it works

Reply:
Yes, this was checked in by accident.